client/finality-grandpa/rpc/src/lib.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! RPC API for GRANDPA.
+
+#![deny(missing_docs)]
+
+use finality_grandpa::BlockNumberOps;
+use sc_finality_grandpa::{voter, AuthorityId, SharedAuthoritySet, SharedVoterState};
+
+use futures::{FutureExt, TryFutureExt};
+use jsonrpc_derive::rpc;
+use serde::{Deserialize, Serialize};
+use std::{collections::HashSet, fmt::Debug};
+
+const NOT_READY_ERROR_CODE: i64 = 1;
```

Comment:
```suggestion
/// Returned when Grandpa RPC endpoint is not ready.
pub const NOT_READY_ERROR_CODE: i64 = 1;
```

Worth making it public for clients.